api-specifications/retailmedia_2022-01.json

```diff
@@ -4241,7 +4241,8 @@
                     },
                     "message": {
                         "type": "string",
-                        "description": "Failure message, only populated on failure"
+                        "description": "Failure message, only populated on failure",
```

Comment:
`message` is "only populated on failure", apparently `fileSizeBytes`, `rowCount` and `md5Checksum` are only populated on success, shouldn't also set them to nullable?